platforms/aws/giantnetes/variables.tf

```diff
@@ -199,49 +199,27 @@ variable "k8s_api_ip" {
   default     = "172.31.0.1"
 }
 
-variable "subnet_bastion_0" {
-  description = "CIDR for bastion network 0."
-  default     = "10.0.1.0/25"
+variable "subnet_bastion" {
+  description = "CIDR for bastion networks"
+  type        = "list"
+  default     = ["10.0.1.0/25", "10.0.1.128/25"]
 }
 
-variable "subnet_bastion_1" {
-  description = "CIDR for bastion network 1."
-  default     = "10.0.1.128/25"
-}
-
-variable "subnet_elb_0" {
-  description = "CIDR for load balancer network 0."
-  default     = "10.0.2.0/26"
-}
-
-variable "subnet_elb_1" {
-  description = "CIDR for load balancer network 1."
-  default     = "10.0.2.64/26"
-}
-
-variable "subnet_elb_2" {
-  description = "CIDR for load balancer network 2."
-  default     = "10.0.2.128/26"
+variable "subnet_elb" {
+  description = "CIDR for load balancer networks."
+  type        = "list"
+  default     = ["10.0.2.0/26", "10.0.2.64/26", "10.0.2.128/26"]
 }
 
 variable "subnet_vault_0" {
   description = "CIDR for Vault network."
   default     = "10.0.3.0/25"
 }
 
-variable "subnet_worker_0" {
-  description = "CIDR for worker network 0."
-  default     = "10.0.5.0/26"
-}
-
-variable "subnet_worker_1" {
-  description = "CIDR for worker network 1."
-  default     = "10.0.5.64/26"
-}
-
-variable "subnet_worker_2" {
-  description = "CIDR for worker network 2."
-  default     = "10.0.5.128/26"
+variable "subnet_worker" {
```

Comment:
🎉 well done. just a small note. would be better have these variable in plural like `subnet_workers`?

Reply:
Yeah naming is bad, will try to figure out something, so far I like that same thing start with the same word so when ordering  they are  at the same place:
subnets_worker
subnets_bastion
subnets_elb
```